client/src/containers/CompareContainer.js

```diff
@@ -134,6 +156,16 @@ function CompareContainer(props) {
           })}
         </DropDown>
       </DropDownList>
+      {isDataLoaded ? (
```

Comment:
Ideally the isDataLoaded flag is used just once for the entire container and the other tables are grouped accordingly

Reply:
This was my naive attempt at getting the H2H table to be displayed top of the other tables.  Wrapping the tables in a React.Fragment seems to accomplish that without the repeated isDataLoaded use-  does that seem like the right way to handle it?

Reply:
It probably works but you have less styling over it, nesting in a bunch of divs/containers is how I would do it just have to make the naming better. Lots of extras due to the necessary single parent containers 

MainContainer > isDataLoaded > AnotherContainer > [TopContainer[Table1] BottomContainer[Table2 Table3]  ]